docs/api/java/CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ if(BUILD_BINDINGS_JAVA)
       -d ${JAVADOC_OUTPUT_DIR}
       -cp ${CMAKE_BINARY_DIR}/src/api/java/cvc5.jar
       -notimestamp
+    COMMAND find ${JAVADOC_OUTPUT_DIR} -type f -exec sed -i'orig' 's/<!-- Generated by javadoc [^>]* -->//' {} \;
```

Comment:
Sorry, I hadn't tested it. This may work:

```suggestion
    COMMAND find ${JAVADOC_OUTPUT_DIR} -type f -exec sed -i'orig' 's/<!-- Generated by javadoc [^>]* -->//' {} "\;"
```